src/js/intercom.ts

```diff
@@ -1,9 +1,11 @@
+let loggedInUserEmail: string| undefined;
```

Comment:
What's going on here?. Should'nt this be stored in the Redux state like everything else? Also missing space.